docs/basics.rst

```diff
@@ -125,9 +125,8 @@ It is often useful to change the precision during only part of a calculation. A
     >>> # do_something()
     >>> mp.prec -= 2
 
-Since Python 2.5, the ``with`` statement along with the mpmath functions ``workprec``, ``workdps``, ``extraprec`` and ``extradps`` can be used to temporarily change precision in a more safe manner:
+The ``with`` statement along with the mpmath functions ``workprec``, ``workdps``, ``extraprec`` and ``extradps`` can be used to temporarily change precision in a more safe manner:
```

Comment:
You left below yet another remark about the with statement ("The effect of the ``with`` statement can be emulated in Python 2.4...")